Code.gs

```diff
@@ -23,8 +23,10 @@
 
 var sourceCalendars = [                // The ics/ical urls that you want to get events from along with their target calendars (list a new row for each mapping of ICS url to Google Calendar)
                                        // For instance: ["https://www.calendarlabs.com/ical-calendar/ics/76/US_Holidays.ics", "US Holidays"]
+                                       // Or with colors following mapping https://developers.google.com/apps-script/reference/calendar/event-color, 
+                                       // for instance: ["https://www.calendarlabs.com/ical-calendar/ics/76/US_Holidays.ics", "US Holidays", "11"]
   ["icsUrl1", "targetCalendar1"],
-  ["icsUrl2", "targetCalendar2"],
+  ["icsUrl2", "targetCalendar2", "11"],
```

Comment:
I think this is the final request from me - let's remove the "11" in the example array. There's already an example in the comment above and I think that is good enough.

Reply:
Done 🙂